### DISCUSSION TITLE
Minimum Stats

### DISCUSSION THREAD

**Discussion by ChristophHandschuh** [ohmplatform/FreedomGPT#23] [General]
What are the minimum stats your pc must have?

**Comment by ohmguru** [comment]
16GB recommended as Model uses 8GB

**Comment by ItsPi3141** [comment]
- CPU with AVX2 support (not required but strongly recommended)
- 6GB of free RAM (8GB RAM will work fine)
- 4.3GB of storage space on your user profile folder (%userprofile% on windows and ~/ on linux)
- No GPU requirement since it is not used
